.ci/38-MIN.yaml

```diff
@@ -1,17 +1,17 @@
-name: test
+name: py38_spopt
 channels:
   - conda-forge
 dependencies:
   - python=3.8
-  - geopandas=0.10.0
+  - geopandas=0.10
   - libpysal
   - matplotlib
-  - networkx
+  - networkx=2.6
   - numpy=1.21
-  - pandas=1.0.5
-  - scikit-learn=0.22
-  - scipy=1.3.2
-  - shapely
+  - pandas>=0.23.0,!=1.5.0,<2
```

Comment:
I would potentially just pin a specific version here.

Reply:
So, I modeled that after the ["pin" we have in `momepy`](https://github.com/pysal/momepy/blob/fbf2c2fbfc95a133c9cebcb35eb59e169a1bcf5c/ci/envs/38-minimal.yaml#L13). Perhaps we should update that, as well? Here' let's try a [`pandas==1.3`]() to conform with the [SPEC 0 — MINIMUM SUPPORTED VERSIONS](https://scientific-python.org/specs/spec-0000/) you posted. I will update the `pyproject.toml` accordingly.